build/cluster.js

```diff
@@ -0,0 +1,102 @@
+// Copyright 2015 Google Inc.
```

Comment:
There's "All rights reserved" to add. Look at other files. I know that some of files are missing it, but the ultimate goal is to unify codebase. If you see it missing, just add when modifying files.